ts/components/wallet/PayWebViewModal.tsx

```diff
@@ -185,13 +182,7 @@ export const PayWebViewModal = (props: Props) => {
           <View
             style={styles.descriptionContainer}
             testID={"PayWebViewModal-description"}
-          >
-            <InfoBox iconName="info" iconColor="bluegreyDark" iconSize={24}>
-              <Label weight={"Regular"} color={"bluegrey"}>
-                {I18n.t("wallet.challenge3ds.description")}
-              </Label>
-            </InfoBox>
-          </View>
```

Comment:
Is this element not needed anymore?

Reply:
I think we can ignore those changes as the file will be removed in this PR #6210 
@dmnplb what are your thoughts?